### PR TITLE
feat: leave meeting button

### PR DESCRIPTION
**Pull request by Scroody**
### What does this PR do?
Introduces join parameter userdata-bbb_direct_leave_button and also an entry for public.app.defaultSettings.application.directLeaveButton: false such that

if true:
An extra button will appear immediately to the left of the three-dots menu. The icon will be the same as the existing leave icon. The background will be red ('danger'). Clicking on this button will act the same way as clicking on the existing button today in the three dots menu.

If false (or if no such parameter passed) (i.e. default) - the leave button will be in the list of options opened by the three dots button.

###Closes Issue(s)
Closes https://github.com/bigbluebutton/bigbluebutton/issues/12614

As described in the original PR by @ramonlsouza - https://github.com/bigbluebutton/bigbluebutton/pull/19080

PS: Added end meeting tweaks (pr #19261) by @Ithanil.

### CODE REVIEW

**Comment by Ithanil** [comment]
https://github.com/bigbluebutton/bigbluebutton/pull/19232 needs to added to this. Would you like me to take care of that? Or should this be merged first and then I make another PR with the second iteration of changes?

File: bigbluebutton-html5/private/config/settings.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ public:
     askForFeedbackOnLogout: false
     # the default logoutUrl matches window.location.origin i.e. bigbluebutton.org for demo.bigbluebutton.org
     # in some cases we want only custom logoutUrl to be used when provided on meeting create. Default value: true
-    askForConfirmationOnLeave: true
+    askForConfirmationOnLeave: false
```
Comment (ramonlsouza) [inline bigbluebutton-html5/private/config/settings.yml:61]:
```suggestion
    askForConfirmationOnLeave: true
```

**Review by antonbsa — APPROVED** [review]
Manually tested and working well! I've already updated the automated tests and everything is passing!

**Comment by ramonlsouza** [comment]
closes #19490

**Pull request merged**